apps/s_client.c

```diff
@@ -1554,6 +1558,15 @@ int s_client_main(int argc, char **argv)
         }
     }
 
+    if (local) {
+        if (!connectstr || ! (host && port)) {
```

Comment:
please use explicit tests against NULL for pointers.

Reply:
Bleurgh, Ok, I'll make the code wordier and less readable if that's the house style :-)

Reply:
And, of course, the way I wrote it was wrong!